packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -97,14 +120,18 @@ ListEntries.propTypes = {
 	highlightKey: PropTypes.string,
 	/** The data for the list. */
 	list: PropTypes.array,
+	/** Whether items are reorderable. */
+	reorderable: PropTypes.bool,
 	/** Whether items are listed as columns. */
 	column: PropTypes.bool,
-	/** Function invoked to get each item's key. */
+	/** Function invoked to get each item's key. Defaults to true if reorderable */
```

Comment:
I think this change was meant for one line above? It sounds like it is describing what happens to the `column` prop